chrome/content/zotero/elements/itemBox.js

```diff
@@ -2313,6 +2313,19 @@
 		focusFirstField() {
 			this._focusNextField(1);
 		}
+
+		focusLastField() {
+			const tabbableFields=this.querySelectorAll('*[ztabindex]:not([disabled=true])');
+			const last=tabbableFields[tabbableFields.length-1];
```

Comment:
Make sure you have eslint set up. You removed spaces around the `=` and `-`, which should be flagged by eslint. (Lots of existing warnings in existing code, but new code should be correct.)